xenial/2.3.6/Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM ubuntu:xenial
 
-MAINTAINER Daniel Alan Miller <dalanmiller@rethinkdb.com>
+MAINTAINER Gabor Boros <gabor.brs@gmail.com>
+ENV RETHINKDB_PACKAGE_VERSION 2.3.6~0xenial
```

Comment:
This should be as low in the Dockerfile as possible so that Docker build cache can be reused as much as possible. So just before line 15 of the new file:

```Dockerfile
RUN apt-get update -qq \
￼	&& apt-get install -y rethinkdb=$RETHINKDB_PACKAGE_VERSION \
```

Reply:
As far as I can see `ENV ` is cached too and the docker cache can reuse it, or am I missing something?

<img width="597" alt="Screenshot 2019-11-29 at 6 44 51" src="https://user-images.githubusercontent.com/19173947/69846328-f047d580-1273-11ea-9ad8-2a5e9f3bffe7.png">

Reply:
Yes it’s cached, but it will invalidate the following caches when you change the version (however that is probably a good thing in this case as the apt-update should be forced anyway.